src/lib/containers/entity_finder/details/view/DetailsView.tsx

```diff
@@ -57,149 +98,380 @@ export const DetailsView: React.FunctionComponent<DetailsViewProps> = ({
-                <div>
-                  <span>
-                    Version
-                    <HelpButtonPopover
-                      contentMarkdown={
-                        'Allows you to choose which version of this item you would like to perform this action on. If you would like the selected reference to update as new versions are created, choose “Always Latest Version”'
-                      }
+    <div className="EntityFinderDetailsView bootstrap-4-backport">
+      <BlockingLoader show={showLoadingScreen} />
+      <AutoResizer className="DetailsViewAutosizer" onResize={rebuildTooltip}>
+        {({ height, width }: { height: number; width: number }) => (
+          <BaseTable
+            classPrefix="DetailsViewTable"
+            data={tableData}
+            height={height}
+            width={autoSizeWidth ? width : 1200}
+            rowHeight={46}
+            overscanRowCount={5}
+            // Apply classes to the rows for styling
+            rowClassName={({ rowIndex }: { rowIndex: number }) => {
+              let className = 'EntityFinderDetailsViewRow'
+              if (rowIndex % 2 === 0) {
+                // Apply a class based on index so we can get alternating colors
+                // We don't use CSSs nth-child because the rows are virtualized, so an even child might change to odd on-the-fly
+                className += ' isEven'
+              }
+              return className
+            }}
+            // Apply aria roles to the rows for a11y/styling
+            rowProps={({ rowData }) => {
+              return {
+                'aria-selected': rowData.isSelected,
+                'aria-disabled': rowData.isDisabled,
+                'aria-hidden': rowData.isHidden,
+              }
+            }}
+            headerCellProps={{
+              role: 'columnheader',
+            }}
+            // Sorting:
+            sortState={sortState}
+            components={{ SortIndicator: CustomSortIndicator }}
+            onColumnSort={({ key, order }) => {
+              if (sort && setSort) {
+                setSort(
+                  key as SortBy,
+                  order === 'asc' ? Direction.ASC : Direction.DESC,
+                )
+              }
+            }}
+            onScroll={rebuildTooltip}
+            rowEventHandlers={{
+              onClick: ({ rowData }) => {
+                const { id, isDisabled, isVersionableEntity } = rowData
+                let { currentSelectedVersion } = rowData
+                if (!isDisabled) {
+                  if (
+                    isVersionableEntity &&
+                    mustSelectVersionNumber &&
+                    currentSelectedVersion == null &&
+                    Object.prototype.hasOwnProperty.call(
+                      rowData,
+                      'versionNumber',
+                    )
+                  ) {
+                    currentSelectedVersion = (rowData as EntityHeader)
+                      .versionNumber
```

Comment:
Comment why we don't need to handl;e the case where we dont have version -- VersionRenderer will take care of it